interface/main/tabs/js/user_data_view_model.js

```diff
@@ -33,10 +33,10 @@ function user_data_view_model(username,fname,lname,authGrp)
 
 }
 
-function viewPtFinder(myMessage, searchAnyType, event)
+function viewPtFinder(myMessage, searchAnyType, data, event)
```

Comment:
Um why is data placed here?

Reply:
no idea, but it definitely is not an event :)
After reading through following it will become a little more clear (from a dense fog to a near dense fog :) ):
https://knockoutjs.com/documentation/click-binding.html

Reply:
wooohooo, I finally found specific confirmation of why this works:
https://stackoverflow.com/questions/35921798/passing-the-event-object-to-knockout-js-function
data is the current binding context (whatever the heck that means)
and bind basically combines the parameters and places them before the data, event parameters

Gonna bring this in and then try to forget about this :)